framework/include/base/Moose.h

```diff
@@ -9,9 +9,11 @@
 
 #pragma once
 
+#include "libMeshReducedNamespace.h"
 #include "libmesh/perf_log.h"
 #include "libmesh/libmesh_common.h"
 #include "XTermConstants.h"
+#include "libmesh/utility.h" // to declare namespace
```

Comment:
self-note to remove this

Reply:
You want it just for the namespace declaration?  You can just put `namespace libMesh {}` in MOOSE code to get that.

Reply:
yup! I ended up figuring this out later on